backend/source/master/procurement_questions.csv

```diff
@@ -0,0 +1,21 @@
+Question,Select answer,Description
```

Comment:
Hi mas @ifirmawan, 
What if we add an ID column to the CSV file and also include this ID in the seeder?
This way, we can use the ID when updating questions without needing to truncate them.

Reply:
Hi Mas @wayangalihpratama  Thank you for your review.
Since question id and choices are not included in the formula calculation, I don't think we need to define those additional columns.